test/response_test.rb

```diff
@@ -363,29 +328,24 @@ class RubySamlTest < Minitest::Test
 
     describe "#session_expires_at" do
       it "extract the value of the SessionNotOnOrAfter attribute" do
-        response = OneLogin::RubySaml::Response.new(response_document)
         assert response.session_expires_at.is_a?(Time)
 
-        response = OneLogin::RubySaml::Response.new(response_document_2)
-        assert_nil response.session_expires_at
+        assert_nil response_without_attributes.session_expires_at
```

Comment:
imo, this should go in a different test case, stating something like: `it "returns nil when the response does not have attributes"`